diagnostic.md

```diff
@@ -32,23 +32,23 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+Getter methods: Selecting by .class or #id. Setter methods .find() .text()
```

Comment:
//getters
.val()
.html()
.text()
.css('<some property>')
.attr('<some attribute>')

//setters
.val('some text')
.html('some html')
.text('some text')
.css('<some property>', '<some value>')
.attr('<some attribute>', '<some value>')

//this is not a complete list of getters and setters.